source/index.md

```diff
@@ -1,103 +1,57 @@
-  <img alt="OpenFisca logo" class="logo-margin" src="https://openfisca.org/img/logo-openfisca.svg">
-</a>
+[OpenFisca](https://www.openfisca.org) is an open source project that allows you to turn legislation into code.
 
-<div class="download-text">
-Get an offline version of this documentation.
-</div>
-<div class="button-download"><a class="button-appearance" href="https://media.readthedocs.org/pdf/openfisca/latest/openfisca.pdf"><i class="fas fa-download"></i> Documentation</a></div>
+Describe your tax & benefit system, provide a situation as input (i.e income), ask for a calculation as output (i.e. income tax), and get your results.
 
-<div class="header-end"></div>
+* **For economists**: OpenFisca allows you to use survey data to simulate the impact of a  reform on a given government’s budget and on a population’s standard of living.
+* **For developers**: OpenFisca allows you to easily create web applications based on your simulation results, thanks to the web API. You can build a great variety of other services by coding formulas, hosting your own instance or building your own extensions.
+* **For public administrations**: OpenFisca allows you to stop building your micro-simulation software and tax & benefit calculators on your own. Instead, contribute to OpenFisca, collaborate with other administrations and reduce the bill for the taxpayer.
 
-# Introduction
+## How does OpenFisca work?
 
-[OpenFisca](https://openfisca.org/) transforms legislation into code.
+### 1 - Choose an available tax & benefit system or roll your own:
 
-OpenFisca allows you to:
+With OpenFisca, you can:
+* Use an existing tax & benefit system (see the [list of systems already built](https://openfisca.org/en/countries/) )
+* [Build a new tax & benefit system](key-concepts/tax_and_benefit_system.html) if it doesn’t exist already
+* [Contribute](/contribute/) to an existing system by adding or improving elements of the legislation
 
-- Calculate many variables of the tax and benefit system of a country given input variables.
-> OpenFisca can calculate social benefits and taxes on test cases (a person or a household).
-- Simulate the budgetary consequences of a reform and its distributional impact when plugged on a survey.
-> OpenFisca can calculate social benefits and taxes on population data (real data or survey data).
+How to turn legal code into Python code?
 
-Its engine is independent of the country, it is therefore possible to simulate any country. It behaves as [microsimulation](https://en.wikipedia.org/wiki/Microsimulation) software with improved ties to legislation.
+* First, identify some legislation that can be expressed as an arithmetic operation
+* More tech than legislation savvy? Take a look at some [recipes](/coding-the-legislation/10_basic_example.html) to easily communicate with expert of the law.
+* Then, try to translate them into [formulas, variables, parameters, etc.](/key-concepts/variables.html)
```

Comment:
Translating laws into OpenFisca variables and parameters is actually the topic of the "From law to code" section. 

What about:

```suggestion
* Then, try to translate them into [formulas, variables, parameters, etc.](/coding-the-legislation/10_basic_example.html)
```

?